.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Report bugs with the Microsoft Edge documentation website
```

Comment:
There's still a rather subtle distinction between "issue w/ the docs site" vs. "issue with the docs".  I had to take time to think about the difference; can spell out the contrast more explicitly like in a hallway conversation.
```suggestion
about: Report bugs with the Microsoft Edge documentation website, such as a broken link or Markdown tagging that's not working.
```

Reply:
Move "report documentation content" to be above "documentation bug report".

Reply:
Agreed, we don't need the distinction between these 2 types of issues.